src/otx/core/model/module/action_classification.py

```diff
@@ -15,18 +17,26 @@
 from otx.core.model.entity.action_classification import OTXActionClsModel
 from otx.core.model.module.base import OTXLitModule
 
+if TYPE_CHECKING:
+    from lightning.pytorch.cli import LRSchedulerCallable, OptimizerCallable
+
 
 class OTXActionClsLitModule(OTXLitModule):
     """Base class for the lightning module used in OTX detection task."""
 
     def __init__(
         self,
         otx_model: OTXActionClsModel,
-        optimizer: torch.optim.Optimizer,
-        scheduler: torch.optim.lr_scheduler.LRScheduler,
         torch_compile: bool,
+        optimizer: OptimizerCallable = lambda p: torch.optim.SGD(p, lr=0.01),
+        scheduler: LRSchedulerCallable = torch.optim.lr_scheduler.ConstantLR,
     ):
-        super().__init__(otx_model, optimizer, scheduler, torch_compile)
+        super().__init__(
+            otx_model=otx_model,
```

Comment:
This is currently associated with the OTXLitModule in core, so i expect it to change a lot to work with the current PR. It would be nice if we remembered this and renamed it to the appropriate word at once, just like the others.